File: frontend/User_Mobile_flutter/lib/models/foodCatalogResponse.dart
class FoodCatalogResponse {
  int foodCatalogId;
  String foodCatalogName;
  String foodCatalogDescription;

  FoodCatalogResponse(
    this.foodCatalogId,
    this.foodCatalogName,
    this.foodCatalogDescription,
  );

  factory FoodCatalogResponse.fromJson(Map<String, dynamic> json) {
    return FoodCatalogResponse(
      json['foodCatalogId'],
      json['foodCatalogName'],
      json['foodCatalogDescription'],
    );
  }
}

File: backend/be/src/main/java/nhom27/itss/be/dto/response/FoodCatalogResponse.java
package nhom27.itss.be.dto.response;

import lombok.*;
import lombok.experimental.FieldDefaults;

import java.sql.Timestamp;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
@Getter
@FieldDefaults(level = AccessLevel.PRIVATE)
public class FoodCatalogResponse {
    Integer foodCatalogId;
    String foodCatalogName;
    String foodCatalogDescription;

}
